src/OpenTelemetry.ResourceDetectors.Container/KubernetesProperties.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace OpenTelemetry.ResourceDetectors.Container;
+
+internal static class KubernetesProperties
+{
+    public const string KubernetesPortEnvVar = "KUBERNETES_PORT";
+    public const string KubernetesServiceHostEnvVar = "KUBERNETES_SERVICE_HOST";
+    public const string KubernetesServicePortEnvVar = "KUBERNETES_SERVICE_PORT";
+
+    public const string HostnameEnvVar = "HOSTNAME";
+
+    public const string ContainerNameEnvVar = "CONTAINER_NAME";
+    public const string ContainerNameEnvVar2 = "container.name";
```

Comment:
The name of this env variable should probably be `k8s.container.name` to match the spec: https://opentelemetry.io/docs/specs/semconv/resource/k8s/#container. And probably `K8S_CONTAINER_NAME` for the other.